src/xrGame/stalker_animation_manager.h

```diff
@@ -69,8 +69,8 @@ class CStalkerAnimationManager
     CMissile* m_missile;
 
 private:
-    bool m_call_script_callback;
-    bool m_call_global_callback;
+    bool m_call_script_callback{false};
+    bool m_call_global_callback{false};
```

Comment:
For braces, we always insert spaces inside them.
```suggestion
    bool m_call_script_callback{ false };
    bool m_call_global_callback{ false };
```